src/Groups/perm.jl

```diff
@@ -286,7 +286,9 @@ function cperm()
 end
 
 function cperm(L1::AbstractVector{T}, L::AbstractVector{T}...) where T <: IntegerUnion
-  return prod([PermGroupElem(symmetric_group(maximum(y)), GAPWrap.CycleFromList(GAP.Obj([Int(k) for k in y]))) for y in [L1, L...]])
+  cycles = [L1, L...]
+  n = maximum(map(maximum, cycles))
+  return prod([PermGroupElem(symmetric_group(n), GAPWrap.CycleFromList(GAP.Obj([Int(k) for k in y]))) for y in cycles])
```

Comment:
Argh, this reminds me I really need to submit a PR for my rewrite of `cperm` and friends.


Also, this method is still inefficient because it creates a fresh `symmetric_group` each time. Either we cache those (see issue #129) or perhaps we should just disallow this without an explicit parent group (well, not immediately to stay compatible with 1.0, but at least we could deprecate this). Thoughts, @fieker ?